src/api2/ComputeEngineEmpty.java

```diff
@@ -1,12 +1,14 @@
 package api2;
 
+import java.util.List;
+
 public class ComputeEngineEmpty implements ComputeEngine {
     private InputConfig input ;
     public ComputeEngineEmpty (InputConfig input){
         this.input = input;
     }
     @Override
-    public String compute(int input) {
+    public String compute(List<String> input) {
         throw new UnsupportedOperationException("Unimplemented method 'compute'");
     }
     
```

Comment:
When you are making the computation, make sure the variables align with DataStore and Coordinator